### PR TITLE
Sage tweaks

### PR DESCRIPTION
**Pull request by RawKStar77**
Restoring somethings that were lost during the upgrade to sage, and bringing the theme to a state where we can use.

Thanks @tilgovi for your efforts!

### CODE REVIEW

File: templates/footer.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <footer class="content-info" role="contentinfo">
   <div class="container">
-    <?php dynamic_sidebar('sidebar-footer'); ?>
+    <?php wp_nav_menu( array('container_class' => 'menu-footer', 'menu_class' => 'nav navbar-nav', 'theme_location' => 'secondary_navigation') ); ?>
```
Comment (tilgovi) [inline templates/footer.php]: This isn't necessary and I think we should remove it. It's a deviation from upstream when upstream has "widgets" and among them is a way to set the footer to a custom menu. I had this working fine.

File: templates/head.php [hunk 2]
```diff
@@ -5,5 +5,6 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="alternate" type="application/rss+xml" title="<?= get_bloginfo('name'); ?> Feed" href="<?= esc_url(get_feed_link()); ?>">
+    <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
```
Comment (tilgovi) [inline templates/head.php]: We can avoid this by importing it in our less instead of in our template.

File: fullwidth-page.php [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<?php
+/*
+Template Name: Full Width Page
+*/
+?>
+
+<div class="container hypo-space-at-bottom">
```
Comment (tilgovi) [inline fullwidth-page.php]: hypo-space-at-bottom is gone

**Comment by tilgovi** [comment]
Made a couple comments if you want to take them. I hope I didn't step on any toes.

**Comment by tilgovi** [comment]
I was trying as hard as possible to stay away from modifying upstream files so that we can continue to pull updates from them with as few merge conflicts as possible.

File: templates/footer.php (hunk 1 above)
Comment (RawKStar77) [inline templates/footer.php]: Ah, sorry, I didn't realize.

**Comment by RawKStar77** [comment]
> Made a couple comments if you want to take them. I hope I didn't step on any toes.

Not at all. : ) Thanks for helping to update this.

**Pull request merged**